USFMToolsSharp.Renderers.Docx/DocxRenderer.cs

```diff
@@ -281,7 +303,51 @@ public void createBookHeaders(string bookname)
 
             bookNameCount++;
         }
-
+        public void getRenderedRows(Marker input, StyleConfig config,XWPFTable parentTable)
+        {
+            XWPFTableRow tableRowContainer = parentTable.CreateRow();
+            foreach (Marker marker in input.Contents)
```

Comment:
I'm fairly certain the cells need to be rendered in they're specified order but don't worry about that for the time being.

Reply:
@rbnswartz Are you referring to the Column position?